frontend/sample.env

```diff
@@ -1,3 +1,5 @@
 export NEXT_PUBLIC_MIXPANEL_TOKEN="<YOUR MIXPANEL TOKEN HERE>"
 export NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY="<stripe publishable key>"
-export NEXT_PUBLIC_MOONSTREAM_API_URL=http://localhost:7481
\ No newline at end of file
+export NEXT_PUBLIC_MOONSTREAM_API_URL="<moonstream_api_url>"
+export NEXT_PUBLIC_MOONSTREAM_CRAWLERS_URL="<moonstream_crawlers_url>"
```

Comment:
This is really `NEXT_PUBLIC_MOONSTREAM_ETHEREUM_CLUSTER_URL`. We are moving crawlers to different machine.